### PR TITLE
keystone: revert back certificate attributes setup

### PR DESCRIPTION
**Pull request by sidthe**
Due to the fact that Ceph depends on Keystone signing certificates for communication with Keystone we need to have the right code to set up the attributes and related content.

relates to this https://github.com/crowbar/crowbar-ceph/pull/83

### CODE REVIEW

**Comment by sidthe** [comment]
needed for https://github.com/crowbar/crowbar-ceph/pull/83

**Comment by sidthe** [comment]
I don't see a way to optimize the block size and usage of blocks across multi lines, so I will be ignoring that. This was supposed to be a partial code revert.

**Comment by sidthe** [comment]
[(3) Remove support for PKI tokens](https://trello.com/c/RJGh4Jg1/394-3-remove-support-for-pki-tokens)

File: chef/cookbooks/keystone/recipes/server.rb [hunk 1]
```diff
@@ -286,6 +286,84 @@
 
 crowbar_pacemaker_sync_mark "create-keystone_db_sync"
```
Comment (vuntz) [inline chef/cookbooks/keystone/recipes/server.rb]: You're missing this bit here:
```ruby
# Make sure the certificates code is run on the founder first
crowbar_pacemaker_sync_mark "wait-keystone_certificates" do
  fatal true
end
```

Comment (sidthe) [inline chef/cookbooks/keystone/recipes/server.rb]: indeed, thx!

**Comment by aplanas** [comment]
Actually I agree with @vuntz to do a proper git revert here. Usually helps.

**Comment by sidthe** [comment]
@aplanas if it is not critical to you I would not do it, as I already did some changes here and there, so as long as this is critical I'd just live it the way it is

**Comment by aplanas** [comment]
@sidthe I do not think that is critical, sure.

**Pull request merged**